gemfiles/rails4.0.gemfile.lock

```diff
@@ -1,60 +0,0 @@
-PATH
```

Comment:
They are already ignored. I'm just removing the files since they're in this weird state where ignored but still exist.